### ISSUE TITLE
Create sample map and send map data to client

### ISSUE THREAD

**Issue by borrascador** [borrascador/rain#279]
We might already have a sufficient sample map in `src/server/map/map.json`. If not, we need to generate a sample map with at least two supertiles.

Once we have the map integrated into the server, the data structure must be sent to the client.

**Comment by amos1814** [comment]
map/map.json doesn't have the correct values for x, y, position. I should be able to fix that in half an hour at most. Adding reasonable road and water data for testing might take 2 or 3 hours.

**Comment by borrascador** [comment]
I kind of feel like we should create a separate story for adding the road and water stuff. Getting a working map is all I need to get unblocked.
